tls/s2n_config.c

```diff
@@ -562,6 +571,45 @@ static int s2n_config_add_cert_chain_and_key_impl(struct s2n_config *config, str
     return S2N_SUCCESS;
 }
 
+S2N_RESULT s2n_config_validate_certificate_preferences(const struct s2n_config *config,
```

Comment:
```suggestion
S2N_RESULT s2n_config_validate_stored_certificates(const struct s2n_config *config,
```
Do you think this name makes it more clear what this function is doing?

Reply:
Renamed to `s2n_config_validate_loaded_certificates`